src/custom/pages/Profile/index.tsx

```diff
@@ -1,87 +1,148 @@
+    label += `${mockTime.getMinutes()} ${mockTime.getMinutes() > 1 ? 'mins' : 'min'}`
+  } else if (mockTime.getSeconds() > 0) {
+    label += `${mockTime.getSeconds()} ${mockTime.getSeconds() > 1 ? 'secs' : 'sec'}`
+  } else {
+    label = 'Just now'
   }
-`
-
-const TextGroup = styled.div`
-  margin-left: 1.5rem;
-`
-
-const Wrapper = styled(Page)`
-  ${GdocsListStyle}
-
-  max-width: 1000px;
-  padding-top: 1rem;
-
-  span[role='img'] {
-    font-size: 1.8em;
+  const ethAddress = 'your-ethereum-address'
+  const copyToClipboard = () => {
+    alert('Copied!')
   }
-`
-
-export default function Profile() {
   return (
     <Wrapper>
-      <Content>
-        <FlexContainer>
-          <div>
-            <Title>Profile</Title>
-            <TextGroup>
-              <strong>
-                <span role="img" aria-label="Milk">
-                  🥛
-                </span>
-                Milk taste better, when shared with friends!
-              </strong>
-              <p>
-                You love CowSwap, we know that, but what if we told you you can let others love it too and get some
-                rewards for doing so.
-              </p>
-              <br />
-              <p>Join CowSwap affiliate program, and start now accruing rewards every time they trade.</p>
-              <p>
-                The best part, is your referrals will also get rewards for trading if they join CowSwap using your link.
-              </p>
-            </TextGroup>
-          </div>
-          <img src={CowsImg} alt="Cows" style={{ marginBottom: '1rem' }} />
-        </FlexContainer>
-
-        <b style={{ marginLeft: '1.5rem' }}>Create your referral link now:</b>
-        <ButtonGroup>
-          <ButtonContainer>
-            <ButtonPrimary>
-              <Trans>Create affiliate link</Trans>
-            </ButtonPrimary>
-          </ButtonContainer>
-          <ButtonContainer>
-            <ButtonSecondary>
-              <Trans>Learn about the Affiliate Program</Trans>
-            </ButtonSecondary>
-          </ButtonContainer>
-        </ButtonGroup>
-      </Content>
+      <Wrap flexDir={'column'}>
+        <Wrap width={'100%'} alignItems={'center'} grow={'1'} flexDir={['column', 'row']}>
+          <Wrap justifyCont={['center', 'flex-start']} grow={'1'}>
+            <Txt fw={'bold'} txtAlign={['center', 'left']} fs={[18, 26]}>
+              Profile overview
+            </Txt>
+          </Wrap>
+          <Wrap justifyCont={['center', 'flex-end']} grow={'1'}>
+            <RollBackIcon />
+            &nbsp;
+            <Txt>
+              <Txt secondary>Last updated:&nbsp;</Txt>
+              <Txt fw={'bold'}>{label} ago</Txt>
+            </Txt>
+          </Wrap>
+        </Wrap>
+        <StyledWrap>
+          <Txt secondary fw={'bold'}>
+            Your referral url
+          </Txt>
+          <Wrap alignItems={'center'} justifyCont={'center'}>
+            <Txt onClick={() => copyToClipboard()} fs={14} txtAlign={'center'}>
```

Comment:
There's a `<Copy>` component for that you can use. 
It's https://github.com/gnosis/cowswap/blob/7500644d17d8021fed99f218abf6582fde2ac7a1/src/custom/components/Copy/CopyMod.tsx#L31. 
Import it as `import Copy from 'components/Copy'`

Reply:
applied